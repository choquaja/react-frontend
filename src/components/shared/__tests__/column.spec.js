import React from 'react';
import { shallow } from 'enzyme';
import Column from '../column';

describe('Column', () => {
  it('renders without crashing', () => {
    shallow(<Column />);
  });

  it('renders with size prop', () => {
    const component = shallow(<Column size />);
    expect(component).toMatchSnapshot();
  });

  it('renders with getFlex prop', () => {
    const component = shallow(<Column flex />);
    expect(component).toMatchSnapshot();
  });
});
