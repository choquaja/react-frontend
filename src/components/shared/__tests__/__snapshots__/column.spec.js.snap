// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders with getFlex prop 1`] = `
<div
  className="column__Column-ioyXO bXVoGi"
/>
`;

exports[`Column renders with size prop 1`] = `
<div
  className="column__Column-ioyXO lneHDN"
  size={true}
/>
`;
